src/widget.js

```diff
@@ -161,7 +161,7 @@ Widget.prototype = {
       }
 
       let currentStateClass = this.rsWidget.className.match(/rs-state-(.+)(\s|$)/)[0];
```

Comment:
This regex's `(\s|$)` was filtering the class name from the space list. I'm not sure why it doesn't work for you, but I never saw this failing after implementing it like this explicitly.

Reply:
Could it be that I committed it wrong, and the match is actually `[1]` instead of `[0]`?

Reply:
No, the `(.+)` was matching everything, including whitespaces. So it included all other classnames that came after `rs-state-`.